src/main/java/kobe/Ui.java

```diff
@@ -0,0 +1,47 @@
+package kobe;
+
+public class Ui {
```

Comment:
Here the usage of indentation and line seems to be more correct and sensible. However, as i recall, it should not be named "line" and "line2". I mean there is number in it. Should be something without number and more expressive.